src/main/java/org/truffleruby/core/rope/TruffleRopesNodes.java

```diff
@@ -164,6 +165,19 @@ protected RubyString flattenRope(Object string,
 
     }
 
+    @CoreMethod(names = "native_rope", onSingleton = true, required = 1)
```

Comment:
Maybe it should be called `to_native` or so? It doesn't return a Rope (that's not a valid interop object).

Reply:
`convert_to_native` as currently is also fine